src/scripts/main.js

```diff
@@ -1,3 +1,72 @@
 'use strict';
 
-// write code here
+let clickedColumn;
+
+const tableArray = [];
+
+function toNumber(numberAsString) {
+  let resultString = numberAsString;
+
+  if (numberAsString[0] === '$') {
+    resultString = numberAsString.slice(1);
+
+    if (typeof (parseInt(resultString.split(',').join('')))
+    === 'number') {
+      resultString = parseInt(resultString.split(',').join(''));
+    }
+  } else if (parseInt(resultString)) {
+    resultString = parseInt(numberAsString);
+  }
+
+  return resultString;
+}
```

Comment:
use Number() or Unary Operator (+) instead of this funciton
https://dev.to/sanchithasr/7-ways-to-convert-a-string-to-number-in-javascript-4l

Reply:
Number() or Unary Opertor doesn't work with such string as $162,700... I can use them only for Age column...
I can shorten toNumber function to use it only with string similar to $162,700. But in this case it would be a lot of checks in other parts of the code.
Maybe, I've understood something incorrect.